applications/unity-ds/7.7.0/metadata.json

```diff
@@ -11,13 +11,25 @@
    ],
    "Category": "ds",
    "IamRoles": {},
-   "Package": "https://github.com/unity-sds/unity-data-services/releases/download/v7.7.0-merge-9294125742/terraform_marketplace-7.7.0-merge-9294125742.zip",
+   "Package": "https://github.com/unity-sds/unity-data-services/releases/download/v7.7.0-merge-9373961021/terraform_marketplace-7.7.0-merge-9373961021.zip",
    "ManagedDependencies": {},
    "Backend": "terraform",
    "WorkDirectory": "/",
    "DefaultDeployment": {
     "Variables": {
-      "some_terraform_variable": "See variables.tf in package (zip)"
+        "Values": {
+            "project": "",
```

Comment:
these might get inserted automatically to terraform, by placing them here, i _think_ the user can overwrite them. it's probably "ok" for now, but in the future we probably don't need everything in here, just things that a user should be updating.

Reply:
actually, the only item needed might be the bucket name. everything else is referencing known values (SSM or otherwise)

Reply:
Yea. we can hide some of them.. but i'm afraid it might break at this stage if we hide things like `ssm_path`.